scripts/react-native-helper.js

```diff
@@ -67,9 +67,11 @@ module.exports = {
     let command = `npm install @bugsnag/react-native@${version}  --registry ${registryUrl}`
     common.run(command, true)
 
-    // Install navigation tracker
+    // Install navigation trackers
     command = `npm install @bugsnag/plugin-react-navigation@${version} --registry ${registryUrl}`
     common.run(command, true)
+    command = `npm install @bugsnag/plugin-react-native-navigation@${version} --registry ${registryUrl}`
+    common.run(command, true)
```

Comment:
It might be safer to put these in the `install.sh` scripts, just in case something conflicts with having both installed